students/wachulec_maria/lesson_06_dicts_tuples_sets_args_kwargs/access_rights.py

```diff
@@ -0,0 +1,15 @@
+def access_rights():
+    operations = {'write': 'W', 'read': 'R', 'execute': 'X'}
+    files = {}
+    for i in range(int(input())):
```

Comment:
minor convention thing.
`i` is not used inside for loop, so by convention should use `_`
https://stackoverflow.com/questions/5893163/what-is-the-purpose-of-the-single-underscore-variable-in-python
https://hackernoon.com/understanding-the-underscore-of-python-309d1a029edc